devdocs/orientation-guide.md

```diff
@@ -0,0 +1,118 @@
+
+    If you plan to contribute code, continue with the tasks below.
+
+
+
+_**Phase B**: Set up locally_
+
+1.  Set up TEAMMATES development environment on your computer.  
+    Important:  
+    * Follow instructions to the letter. _Install the specified versions of the tool stack, not the latest versions._  
+    * If you encounter any problems, refer to the [Developer Troubleshooting Guide](https://www.google.com/url?qhttps://docs.google.com/document/d/1_p7WOGryOStPfTGA_ZifE1kVlskb1zfd3HZwc4lE4QQ/pub?embedded%3Dtrue&sa=D&ust=1458427634398000&usg=AFQjCNGVBlVtbfAZEnGnVwGudarEORsfRw).
+2.  Get dev green. It is OK to proceed to the next phase if you have fewer than 5 failing test cases.
+
+
+
+
+_**Phase C**: Deploy_
+
+1.  Deploy your own copy of TEAMMATES to your staging server.
+
+_**Phase D**: Tinker with the code_
+
+1.  Fork our repo at GitHub
+2.  Create a branch named ‘tinker’
+3.  In that branch, do a small change to the code that makes a visible change to TEAMMATES UI. Preferably, your change should improve the UI in some way, or at least not make it worse.
+4.  Deploy the changed version to your staging server.
+5.  Push your changes to your fork.
+6.  Create a new issue in our issue tracker with the following info  
+    Issue title: ‘New contributor self intro [your name]’    
+    a) the link to your app  
+    b) link to your fork  
+    c) a screenshot of the change you did (as seen on the UI)  
+    d) your real name and a short self-intro
+
+
+_**Phase E**: Start contributing_
+After you complete the above tasks, you are ready to become a contributor. Steps for fixing an issue is explained in the [process](https://github.com/TEAMMATES/repo/blob/master/devdocs/process.md) document. New contributors are advised to start doing an issue labeled d.FirstTimers (but do not do more than one of those), move to other issues labeled as d.Contributors  
+
+
+#### **_Important:_**
+
+Fixing an issue quickly is not the important thing. In fact, issues given to new contributors are ones we already know how to fix. We are more interested to see how you go about fixing the issue. We want to know whether you are systematic and detail-oriented.  
+Take your time to learn and follow the workflow to the letter. Do not skip any steps because you think that step is 'not important'. We are more impressed when you finish an issue in fewer attempts than when you finish it in a shorter time but take many attempts because you were not meticulous enough along the way.  
+
+
+## Contacting us
+
+The best way to contact us is to [post a message in our issue tracker](https://github.com/TEAMMATES/teammates/issues/new). Our issue tracker doubles as a discussion forum. You can use it for things like asking questions about the project or requesting technical help.
+
+Alternatively (less preferred), you can email us at _**teammates[at]comp.nus.edu.sg**_
```

Comment:
`[at]` -> `@`. Also, the email doesn't need to be in italic if it's already in bold.